module/VuFind/src/VuFind/ILS/Driver/XCNCIP2.php

```diff
@@ -44,6 +46,7 @@ class XCNCIP2 extends AbstractBase implements \VuFindHttp\HttpServiceAwareInterf
 {
     use \VuFindHttp\HttpServiceAwareTrait;
     use \VuFind\Log\LoggerAwareTrait;
+    use \VuFind\ILS\Driver\CacheTrait;
```

Comment:
You should add a getCacheKey method that makes sure any cached values are unique for the driver instance.

Reply:
To clarify: This wouldn't normally be an issue unless you use the MultiBackend driver to access two or more XCNCIP services, but it makes sense to prepare for that in any case.

Reply:
I understand. I added getCacheKey method now